README.md

```diff
@@ -1,16 +1,20 @@
-# Title
+# Introduction to Configuring Ruby Applications
 
-## Learning Goals
+We have a number of outside resources we leverage in working with in our Ruby
+applications. For example, ActiveRecord is a Ruby-specific library, or _gem_. We
```

Comment:
Active Record

Reply:
this brick will come before ORMs and Active Record - The view in AT is not in order of display. Use this view to see the brick order: https://airtable.com/tblFOLNdbjmprG1TC/viwjxiUreCq0I52Vi?blocks=hide

Students won't yet have the context to know ORMs and Active Record yet. Maybe another gem? RSpec may be a good one to point to.